packit_service/worker/jobs.py

```diff
@@ -543,7 +555,7 @@ def is_packit_config_present(self) -> bool:
                 self.event.comment,
                 self.service_config.comment_command_prefix,
             )
-            command = arguments["command"]
+            command = arguments.command if arguments else ""
```

Comment:
Looking at this I wonder if we can figure a more sane default here 👀 I still need to check context in which these variables are used… but I think that there should be a cleaner solution to this than multiple ternaries…

Reply:
Just checked, both `command` and `monorepo_package` can be `None`. It is correctly handled in the functions where they are passed to… so I would propose adjusting the way `ParsedComment` works… to always return an object, i.e., `ParsedComment`, and have defaults for `command` and `monorepo_package` that are `None`. See other suggestions.